test/DynamoCoreTests/Engine/LiveRunnerServicesTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Dynamo.Models;
+using NUnit.Framework;
+using System.IO;
+
+namespace Dynamo.Tests.Engine
+{
+    [TestFixture]
+    class LiveRunnerServicesTest : DynamoModelTestBase
+    {
+        /// <summary>
+        /// This test method will execute the RuntimeMirror GetMirror(string var, bool verboseLogging) method.
+        /// </summary>
+
+        [Test]
+        [Category("UnitTests")]
+        public void LiveRunnerServicesGetMirrorTest()
+        {
+            //Arrange
+            var openPath = Path.Combine(TestDirectory, @"core\DetailedPreviewMargin_Test.dyn");
+            RunModel(openPath);
+            //With this flag set to true we reach the section for logging.
+            CurrentDynamoModel.EngineController.VerboseLogging = true;
+
+            //Act            
+            //This the id of a CoreNodeModels.Input.StringInput node inside the .dyn file
+            var node = CurrentDynamoModel.CurrentWorkspace.NodeFromWorkspace("10d3d247-5378-4186-8f2d-286db0cc61a5");
+            var runtimeMirror = CurrentDynamoModel.EngineController.GetMirror(node.AstIdentifierBase);
+            //CurrentDynamoModel.CurrentWorkspace.up
+
+            //Assert
+            //Verify that the runtimeMirror was created correctly
+            Assert.IsNotNull(runtimeMirror);
+
+        }
+
+        /// <summary>
+        /// This test method will execute the UpdateGraph method from the LiveRunnerServices class
+        /// </summary>
+        [Test]
+        [Category("UnitTests")]
+        public void LiveRunnerServicesUpdateGraph()
+        {
+            //Arrange
+            var openPath = Path.Combine(TestDirectory, @"core\DetailedPreviewMargin_Test.dyn");
+
+            RunModel(openPath);
+            //With this flag set to true we reach the section for logging.
+            CurrentDynamoModel.EngineController.VerboseLogging = true;
+
+            //We cannot set the VerboseLogging flag before the Model is created then in order to execute the UpdateGraph()
+            //we need to call the OpenFileCommand againt with the VerboseLogging already set to true
```

Comment:
spelling (again)

Reply:
@mjkkirschner 
The spelling was fixed as a part or the commit: https://github.com/DynamoDS/Dynamo/pull/10705/commits/b6e00f7d2dbe2c067039bb11bcb164b3bea856c8

Please let me know if you have more comments.
Thanks